docs/docs/playground/getPublicKey.mdx

```diff
@@ -0,0 +1,39 @@
+---
+id: getPublicKey
+title: getPublicKey
+---
+
+#### `getPublicKey()`
+
+<strong>Test Lyra's `getPublicKey` method:</strong>
+
+import { useState } from "react";
+import { getPublicKey } from "@stellar/lyra-api";
+import { PlaygroundInput } from "./basics";
+
+export const GetPublicKeyTest = () => {
```

Comment:
some "magic" happens in mdx. For some reason, you have to export this component even though we're instantiating it right below 🤷

Reply:
Hm defining a component within MDX feels weird? Why not as a .tsx file that gets imported?

Reply:
yeah good thinkin

Reply:
just added a separate folder for the component parts of these mdx files